oak_functions/loader/README.md

```diff
@@ -0,0 +1,84 @@
+# Oak Functions Loader
+
+The _Oak Functions loader_ is responsible for starting the Oak Functions trusted
+runtime, and loading an application with a single Wasm module as the workload.
+The trusted runtime instantiates the given Wasm module for each incoming user
+request, runs the request through it in order to produce a response, and then
+terminates the Wasm instance; each Wasm instance is short lived and cannot
+persist state outside of the request lifetime.
+
+As part of our shift to a distributed runtime, in the future, we may allow
```

Comment:
@conradgrobler Is this paragraph still accurate? Or shall we remove it?

Reply:
This is still part of the future plans, but I don't think this will be implemented in the near future. Also, it is possible that such native instance would actually be implemented as different loaders. So, I think it is fine to remove now and add details in future when we implement this type of support.